modules/nw-multus-advanced-annotations.adoc

```diff
@@ -0,0 +1,202 @@
+You may also reference the pod's `k8s.v1.cni.cncf.io/networks-status` to see which additional network has been
+assigned the default route, by the presence of the `default-route` key in the JSON-formatted list of objects.
+====
+
+
+In order to set a static IP address or MAC address for a pod, the JSON formatted annotations may also be used,
+however, will also require that you create networks that specifically allow this functionality. This can be
+specified in a `rawCNIConfig` for the CNO.
+
+. Edit the CNO CR by running the following command:
++
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+The following YAML describes the configuration parameters for the CNO:
+
+.Cluster Network Operator YAML configuration
+[source,yaml]
+----
+name: <name> <1>
+namespace: <namespace> <2>
+rawCNIConfig: '{ <3>
+  ...
+}'
+type: Raw
+----
+<1> Specify a name for the additional network attachment that you are
+creating. The name must be unique within the specified `namespace`.
+
+<2> Specify the namespace to create the network attachment in. If
+you do not specify a value, then the `default` namespace is used.
+
+<3> Specify the CNI plug-in configuration in JSON format, which
+is based on the following template.
+
+The following object describes the configuration parameters for utilizing static MAC addres
+and IP address using the macvlan CNI plug-in:
+
+.macvlan CNI plug-in JSON configuration object using static IP and MAC address
+[source,json]
+----
+{
+  "cniVersion": "0.3.1",
+  "plugins": [{ <1>
+      "type": "macvlan",
+      "capabilities": { "ips": true }, <2>
+      "master": "eth0", <3>
+      "mode": "bridge",
+      "ipam": {
+        "type": "static"
+      }
+    }, {
+      "capabilities": { "mac": true }, <4>
+      "type": "tuning"
+    }]
+}
+----
+
+<1> The `plugins` field specifies a configuration list of CNI configurations
+
+<2> The `capabilities` key denotes that a request is being made to enable the static IP functionality of a CNI plugins runtime configuration capabilities
+
+<3> The `master` field is specific to the macvlan plugin (TODO: link to see macvlan docs)
+
+<4> Here the `capabilities` key denotes that that a request is made to enable the static MAC address functionality of a CNI plugin
+
+The above network attachment may then be referenced in a JSON formatted annotation, along with keys to specify which
+static IP and MAC address will be assigned to a given pod.
+
+Edit the desired pod with:
++
+----
+$ oc edit pod <name>
+----
+
+.macvlan CNI plug-in JSON configuration object using static IP and MAC address
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {
+        "name": "<name>", <1>
+        "ips": [ "192.0.2.205/24" ], <2>
+        "mac": "CA:FE:C0:FF:EE:00" <3>
+      }
+    ]'
+----
+
+<1> Use the `<name>` as provided when creating the `rawCNIConfig` above.
+
+<2> The desired IP address is provided.
```

Comment:
Should use user provide an IP address? If so, I'd update this to the following: "Provide the desired IP address."

Same with the third option.